apps/opik-backend/src/main/java/com/comet/opik/api/resources/v1/priv/DatasetsResource.java

```diff
@@ -113,10 +113,12 @@ public Response getDatasetById(@PathParam("id") UUID id) {
     public Response findDatasets(
             @QueryParam("page") @Min(1) @DefaultValue("1") int page,
             @QueryParam("size") @Min(1) @DefaultValue("10") int size,
+            @QueryParam("with_experiments_only") boolean withExperimentsOnly,
```

Comment:
Minor: double check the default behaviour. I assume this is false by default (non-nullable). That's the right approach, otherwise the API behaviour would change and affect existing users. If any doubt, make the default value explicit in the query param.

Reply:
Correct. Looking at the previous test, they guarantee that when not specified, the value is false and, therefore, not applied.